src/components/product-search/components/Autocomplete.vue

```diff
@@ -112,6 +132,42 @@ onMounted(() => {
 </template>
 
 <style lang="scss">
+.vs-autocomplete {
+    position: relative;
+
+    button {
```

Comment:
Are we able to use classes here rather than element selectors? Also, can the old SCSS now be removed ?